components/twitter-card.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {fallbackLocale} from './locale'
+
+function TwitterCard({
+  card = 'summary_large_image',
+  title = 'Maintainable CSS with React',
+  description = 'React component styling solved with an elegant API, small footprint, and great performance',
+  image = 'https://cloud.githubusercontent.com/assets/7799266/26611099/74c2a45c-4571-11e7-8290-dbdc822339be.png',
```

Comment:
We should move this image into the `/static` directory and serve from the actual app.

https://github.com/zeit/next.js#static-file-serving-eg-images